src/egain/webChat/client/src/egainServiceDesk.ts

```diff
@@ -0,0 +1,392 @@
+/**
+ * (C) Copyright IBM Corp. 2020.
```

Comment:
When a user has requested an agent and is waiting for an agent to accept the conversation, does your service desk provide any information to the user about how long the wait might be? Does it provide any information about the time the user might have to wait or how many people are ahead of the user? If so web chat does have a mechanism to let you report that information to the user. If your service desk does not then don't worry about it.